docs/source/tracking.rst

```diff
@@ -187,19 +187,54 @@ common tasks:
 Running a Tracking Server
 -------------------------
 
-The MLflow tracking server launched via ``mlflow ui`` also hosts REST APIs for tracking runs,
+The MLflow tracking server launched via ``mlflow server`` also hosts REST APIs for tracking runs,
 writing data to the local filesystem. You can specify a tracking server URI
 via the ``MLFLOW_TRACKING_URI`` environment variable and MLflow's tracking APIs will automatically
 communicate with the tracking server at that URI to create/get run information, log metrics, etc.
 
-For example, to launch a run against a local tracking server, launch ``mlflow ui``, set
-``MLFLOW_TRACKING_URI`` to ``http://localhost:5000``, and run:
+An example configuration for a server is as follows:
+
+.. code:: shell
+
+    mlflow server \
+        --file-store /mnt/persistent-disk \
+        --artifact-root s3://my-mlflow-bucket/ \
+        --host 0.0.0.0
+
+Storage
+^^^^^^^
+There are two properties related to how data is stored:
+
+* ``--file-store`` is where the server will store run and experiment information. This should
+  be a persistent (non-ephemeral) disk.
+* ``--artifact-root`` causes clients to log their artifact output (e.g., models) to this
+  location which is suitable for large data (such as an S3 bucket or shared NFS file system). If
+  you do not provide this option, then clients will write artifacts to `their` local directories,
```

Comment:
Nit: `their` is in code font now, should be `*their*` for italics or `**their**` for bold.

Reply:
My sphinx requires double backtick to go to code font, and this is italics. Will switch to more common one anyway.